docs/json.md

```diff
@@ -179,6 +180,19 @@ To set the `default` for a component of the JSON Schema, the annotation
 "! $default 'value'
 ```
 This also ensures that only components whose value is not equal to a specific default value are serialized to the JSON data file.
+
+Default values can only specified if one of the following rules apply:
```

Comment:
I guess I missed this in my last review, but `be` has vanished
```suggestion
Default values can only be specified if one of the following rules apply:
```

Reply:
This was a regression, again :/